ale_linters/python/mypy.vim

```diff
@@ -2,19 +2,51 @@
 " Description: mypy support for optional python typechecking
 
 call ale#Set('python_mypy_executable', 'mypy')
+call ale#Set('python_mypy_daemon_executable', 'dmypy')
+call ale#Set('python_mypy_daemon_status_file', '.dmypy.json')
 call ale#Set('python_mypy_ignore_invalid_syntax', 0)
 call ale#Set('python_mypy_show_notes', 1)
 call ale#Set('python_mypy_options', '')
+call ale#Set('python_mypy_daemon_options', '--follow-imports=skip')
```

Comment:
This is necessary because unlike plain `mypy`, `dmypy` needs to have the follow-imports flag present. The valid options are `skip` and `error` -- open to suggestions on which we should use.